cumulus_etl/loaders/fhir/bulk_export.py

```diff
@@ -125,7 +149,11 @@ async def export(self) -> None:
         # Finished! We're done waiting and can download all the files
         response_json = response.json()
 
-        self.export_datetime = datetime.datetime.fromisoformat(response_json["transactionTime"])
+        try:
+            transaction_time = response_json.get("transactionTime", "")
+            self.export_datetime = datetime.datetime.fromisoformat(transaction_time)
+        except ValueError:
+            pass  # server gave us a bad timestamp, ignore it :shrug:
```

Comment:
you could? take the system time, which would be pretty close?

Reply:
Well, this isn't a super realistic event in the first place - spec is pretty clear about the format. But also... it's tricky. This timestamp needs to be precise. Spec says:
```
The response SHOULD NOT include any resources modified after this instant, and SHALL include any matching resources modified up to and including this instant. 
```

So if we created a backup timestamp, we couldn't make those same guarantees. I'd rather just not claim anything in this weird edge case. (This timestamp is really only used for the off-by-default completion-tracking feature, where precision is useful.)